study-builder/studies/pancan/emails/consent-complete.tmpl.html

```diff
@@ -0,0 +1,106 @@
+<html>
+  <head>
+    <meta name="format-detection" content="telephone=no,date=no,address=no,email=no,url=no" />
+
+    <title>${text["subject"]}</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700" rel="stylesheet">
+    <style type="text/css">
+      .ExternalClass,
+      .ExternalClass div,
+      .ExternalClass font,
+      .ExternalClass p,
+      .ExternalClass span,
+      .ExternalClass td,
+      img {
+        line-height: 100%;
+      }
+
+      #outlook a {
+        padding: 0;
+      }
+
+      .ExternalClass,
+      .ReadMsgBody {
+        width: 100%;
+      }
+
+      a,
+      blockquote,
+      body,
+      li,
+      p,
+      table,
+      td {
+        -webkit-text-size-adjust: 100%;
+        -ms-text-size-adjust: 100%;
+      }
+
+      table,
+      td {
+        mso-table-lspace: 0;
+        mso-table-rspace: 0;
+      }
+
+      img {
+        -ms-interpolation-mode: bicubic;
+        border: 0;
+        height: auto;
+        outline: 0;
+        text-decoration: none;
+      }
+
+      table {
+        border-collapse: collapse !important;
+      }
+
+      body {
+        height: 100% !important;
+        margin: 0;
+        padding: 0;
+        font-family: 'Roboto', sans-serif;
+      }
+
+      @font-face {
+        font-family: 'Roboto', sans-serif;
+        src: url(https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700);
+        font-weight: 300;
+        font-style: normal;
+      }
+    </style>
+  </head>
+
+  <body style="background: #ffffff; width: 100%; height: 100%; padding-top: 60px; padding-left: 60px;">
+    <table style="width: 540px; padding: 0; -webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%; mso-table-lspace: 0pt; mso-table-rspace: 0pt; font-family: Roboto, sans-serif; border-collapse: collapse !important;" border="0" cellpadding="0" cellspacing="0" width="100%">
+      <tr style="width: 540px;">
+        <td align="left" style="margin: 0; padding: 0; font-family: Roboto, sans-serif; width: 278px; height: 33px">
+          <a href="{{baseWebUrl}}" target="_blank" style="cursor: pointer; text-decoration: none;">
+            <img src="https://storage.googleapis.com/${assetsBucketName}/pancan/logo-count-me-in.png" alt="PanCan Project logo" style="-ms-interpolation-mode: bicubic; border: 0; outline: none; text-decoration: none;" width="250px" height="111px" border="0">
```

Comment:
Should we add translations for `alt` ?

Reply:
As far as I know, we haven't been, but I feel this would be a good idea in any case.